reference.md

```diff
@@ -5,26 +5,141 @@ permalink: /reference/
 
 ## Glossary
 
-FIXME: The glossary would go here, formatted as:
-
-~~~
 {:auto_ids}
-key word 1
-:   explanation 1
 
-key word 2
-:   explanation 2
-~~~
-{: .source}
+adaptive thresholding
+:   
 
-(`{:auto_ids}` is needed at the start
-so that Jekyll will automatically generate a unique ID for each item
-to allow other pages to hyperlink to specific glossary entries.)
-This renders as:
 
-{:auto_ids}
-key word 1
-:   explanation 1
+additive color model
+:   a color model that predicts the appearance of colors by summing the numeric representations of the component colors
+
+
+bacterial colony
+:   a visible cluster of bacteria growing on the surface of or within a solid medium, presumably cultured from a single cell
+
+
+binary image
+:   a digital image that has only two possible values for each pixel. Typically, the two colors used for a binary image are black and white.
+
+
+Gaussian blur
+:   a widely used effect in graphics software, typically to reduce image noise and reduce detail
+
+
+BMP (bitmap image file)
+:  a raster graphics image file format used to store bitmap digital images, independently of the display device 
+
+
+bounding box
+:   the smallest enclosing box for a set of points
+
+
+colorimetrics
+:   the science of describing human color perception
+
+
+channel
+:  the grayscale representation of a primary color in a digital image 
+
+
+contour
+:   
+
+
+child (in contours)
+:   
+
+
+crop
+:  the removal of unwanted outer areas from an image 
+
+
+color histogram
+:  a representation of the number of pixels that have colors in each of a fixed list of color ranges
+
+
+edge detection
+:   a variety of mathematical methods that aim at identifying points in a digital image at which the image brightness changes sharply 
+
+
+fixed-level thresholding
+:   
+
+
+grayscale
+:   an image in which the value of each pixel is a single value representing only the amount of light (or intensity) of that pixel
+
+
+image segmentation
+:   the process of partitioning a digital image into multiple segments to make the image more meaningful and easier to analyze
+
+
+intensity
+:   
+
+
+JPEG
+:  a commonly used method of lossy compression for digital images, particularly for those images produced by digital photography 
+
+
+kernel
+:   a small matrix. It is used for blurring, sharpening, embossing, edge detection, and more
+
+
+left-hand coordinate system
+:   
```

Comment:
Possible definition suggestion:

> Cartesian coordinate system where the positive x, y and z axes point right, up and forward, respectively. Positive rotation is clockwise about the axis of rotation.


From https://www.evl.uic.edu/ralph/508S98/coordinates.html

Reply:
I don't see a license on this page so we can't use the definition word for word. Could you provide a suggested rephrasing?

Reply:
I'm having trouble thinking of a good paraphrase. I could also draw it out, which might be more helpful anyways?